permit_intelligence_demo.py

```
@@ -0,0 +1,330 @@
+                if is_investment_relevant_permit(permit):
+                    relevant_permits.append(permit)
+            
+            return {
+                'total_permits': len(raw_permits),
+                'relevant_permits': relevant_permits,
+                'success': True
+            }
+        else:
+            return {'success': False, 'error': f'API error {response.status_code}'}
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)